share/templates.d/99-generic/runtime-install.tmpl

```diff
@@ -115,8 +117,9 @@ installpkg bridge-utils
 ## hardware utilities/libraries
 installpkg pciutils usbutils ipmitool
 installpkg mt-st smartmontools
-%if basearch != "s390x":
-installpkg hdparm pcmciautils
+installpkg hdparm
```

Comment:
I don't think hdparm is available on s390 so it still needs a wrapper, or --optional